packages/jaeger-ui/src/components/QualityMetrics/BannerText.test.js

```diff
@@ -13,32 +13,38 @@
 // limitations under the License.
 
 import React from 'react';
-import { shallow } from 'enzyme';
-
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
 import BannerText from './BannerText';
 
 describe('BannerText', () => {
   it('renders null when props.bannerText is falsy', () => {
-    expect(shallow(<BannerText />).type()).toBe(null);
+    const { container } = render(<BannerText />);
+    expect(container.firstChild).toBe(null);
   });
 
   it('renders header when props.bannerText is a string', () => {
-    expect(shallow(<BannerText bannerText="foo text" />)).toMatchSnapshot();
+    const { getByText } = render(<BannerText bannerText="foo text" />);
+    const headerElement = getByText('foo text');
+    expect(headerElement).toBeInTheDocument();
   });
 
   it('renders styled header when props.bannerText is a styled value', () => {
-    expect(
-      shallow(
-        <BannerText
-          bannerText={{
-            styling: {
-              background: 'red',
-              color: 'white',
-            },
-            value: 'foo text',
-          }}
-        />
-      )
-    ).toMatchSnapshot();
```

Comment:
Snapshots need to be purged for this too. Otherwise, the tests would pass, but CI will fail.